Test/Unit/Block/JsTest.php

```diff
@@ -318,7 +318,7 @@ public function getSettings()
         $this->assertJson($result, 'The Settings config do not have a proper JSON format.');
 
         $array = json_decode($result, true);
-        $this->assertCount(17, $array, 'The number of keys in the settings is not correct');
+        $this->assertCount(18, $array, 'The number of keys in the settings is not correct');
```

Comment:
this should be made into a global constant to easily keep track of

Reply:
Done